src/content/docs/guides/game design/godot/basics.md

```diff
@@ -0,0 +1,182 @@
+
+- Run the engine in a web browser with the [Godot Web Editor (v4.1.1)](https://editor.godotengine.org/releases/4.1.1.stable). This is the easiest way to get started, but it has [documented limitations](https://docs.godotengine.org/en/stable/getting_started/editor/web_editor.html#limitations) and is not recommended for large projects.
+- Download the android app from the [Google Play Store](https://play.google.com/store/apps/details?id=org.godotengine.editor.v4), while functional it is in early access and may not be stable. It is also difficult to use on phones, and tablets are still difficult without a keyboard and mouse.
+- (Recommended) Download the latest version from the [downloads page](https://godotengine.org/download/) for your platform to run locally. On running, you should see a window similar to this:
+![Godot project manager](./ProjectManager.png)
+
+A version of the engine that uses C# as a programming language is also available. It is not recommended for beginners, but if you are familiar with C# it may be a better option for you. You can download it from the [downloads page](https://godotengine.org/download/) for your desktop platform. However, it is not available for use in the web editor or the android app. 
+
+## Creating a project
+### Making a new project
+Click the **New Project** button to create a new project, the following window will appear:
+![New project window](./NewProject.png)
+
+**Project Name:**
+This is the name of your project, it can be anything you want. For this guide, we will use **Godot Is Awesome!**.
+You can use the **Create Folder** button to create an empty folder with the name of your project at the project path.  
+**Project Path:**
+This is the path to the folder where your project will be stored.
+You can use the **Browse** button to select a folder or type in the path manually.
+It is recommended you use an empty folder for your project.  
+**Renderer:**
+Godot has three renderer presets: Forward+, Mobile, and Compatibility.
+For this guide, we will use the **Forward+** preset.
+Forward+ is only good for desktop and console platforms, and has the most features.
+Mobile is best for mobile platforms, but supports desktop, and has fewer features with better performance.
+Compatibility supports desktop, mobile, and web platforms, it has the fewest features and the best performance. It is not recommended to use compatibility for 3D projects.  
+**Version Control Metadata:**
+This is used to store version control information in the project folder. It is recommended to leave this on Git even if you are not using version control. It is also recommended to use version control for your projects, but it is not required. To use git version contol, after creating the project navigate to the project folder and [create a git repository](https://wiki.tuhuratech.org.nz/guides/git/basics/#creating-a-git-repository).
```

Comment:
We should use local path links instead of including the domain if its on the wiki

Reply:
when using a local path and then clicking on the link it just downloads the .md file